coderd/database/dbauthz/dbauthz.go

```diff
@@ -2052,6 +2059,13 @@ func (q *querier) GetUserCount(ctx context.Context) (int64, error) {
 	return q.db.GetUserCount(ctx)
 }
 
+func (q *querier) GetUserFrobulators(ctx context.Context, userID uuid.UUID) ([]database.Frobulator, error) {
+	if err := q.authorizeContext(ctx, policy.ActionRead, rbac.ResourceFrobulator.WithOwner(userID.String())); err != nil {
+		return nil, err
+	}
+	return q.db.GetUserFrobulators(ctx, userID)
+}
+
```

Comment:
This is correct if `Frobulators` also belong to an organization as well.

Instead `fetchWithPostFilter` should be used. It will fetch all `Frobulators` and filter the returned set with the ones that are authorized.

This does overfetch, so should only be used when the sets are small.